terraform/prod-app.tfvars

```diff
@@ -20,5 +20,6 @@ elb_scheme             = "internet facing"
 elb_load_balancer_type = "application"
 solution_stack_name    = "64bit Amazon Linux 2 v2.3.4 running .NET Core"
 elb_matcher_http_code  = "200"
+cache_port = 7410
```

Comment:
Is there a particular reason we're using different ports in each environment?

Reply:
I'm not saying this needs to be changed, just wondering what the reasoning behind it is. Generally the more similar your different environments are, the less likely you are to run into weird issues in the future.